components/Required/Required.vue

```diff
@@ -1,12 +1,12 @@
 <template>
   <span
-    v-if="required"
+    v-if="props.required"
     class="text-primary"
   >*</span>
 </template>
 
 <script setup lang="ts">
-defineProps<{
+const props = defineProps<{
   required: boolean
 }>()
 </script>
```

Comment:
Why this is needed?

Reply:
According to VS Code, `required` is the function from `useForm` and not the prop